src/python/docs/docstrings/FastLinearRegressor.txt

```diff
@@ -56,7 +56,7 @@
             content/uploads/2016/06/main-3.pdf>`_
     
             `Stochastic Dual Coordinate Ascent Methods for Regularized Loss
-            Minimization <http://www.jmlr.org/papers/volume14/shalev-
+            Minimization <https://www.jmlr.org/papers/volume14/shalev-
```

Comment:
Recommend checking that all the new links work:
```suggestion
            Minimization <http://www.jmlr.org/papers/volume14/shalev-
```

Original works: http://www.jmlr.org/papers/volume14/shalev-shwartz13a/shalev-shwartz13a.pdf